test/testdata/python_gapic_client_pipeline.baseline

```diff
@@ -4,3 +4,5 @@ rm -rf {OUTPUT}/library-v1-gapic-gen-python
 test/fake-repos/toolkit/gradlew -p test/fake-repos/toolkit runVGen -Pclargs=--descriptor_set={OUTPUT}/library-v1.desc,--output={OUTPUT}/library-v1-gapic-gen-python,--service_yaml={CWD}/test/testdata/gapi-example-library-proto/src/main/proto/google/example/library/library.yaml,--gapic_yaml={CWD}/test/testdata/gapi-example-library-proto/src/main/proto/google/example/library/library_gapic.yaml,--gapic_yaml={CWD}/test/testdata/gapi-example-library-proto/src/main/proto/google/example/library/python_gapic.yaml
 test/fake-repos/toolkit/gradlew -p test/fake-repos/toolkit runSynchronizer -Pclargs=--source_path={OUTPUT},--generated_path={OUTPUT}/library-v1-gapic-gen-python,--baseline_path={OUTPUT}/baseline,--auto_merge,--auto_resolve
 gen-api-package --api_name=library/v1 -l python --gax_dir={OUTPUT} --template_root=templates/gax
+devpi use https://gapi-pypi.appspot.com/mvashishtha/dev
+devpi upload --no-vcs --from-dir {OUTPUT}
```

Comment:
Add the newline at the end.
Also, after my commit of #23, this would be --from-dir {OUTPUT}/final